models/gas_dao/ethereum/gas_dao_ethereum_airdrop_claims.sql

```diff
@@ -1,5 +1,6 @@
 {{
     config(
+        tags=['dunesql'],
         schema = 'gas_dao_ethereum',
         alias = alias('airdrop_claims'),
         materialized = 'incremental',
```

Comment:
Same as before, the claim period appears to be over for this airdrop, so it won't need to be incremental

Reply:
I checked the min/max date and there was still an event from the claims table less than two weeks ago so I'm guessing claiming is still ongoing?